pallets/apollo-platform/src/lib.rs

```diff
@@ -120,22 +110,15 @@ pub mod pallet {
 
     pub type AccountIdOf<T> = <T as frame_system::Config>::AccountId;
 
+    /// The current storage version.
+    pub const STORAGE_VERSION: StorageVersion = StorageVersion::new(0);
```

Comment:
Here should be current version
```suggestion
    pub const STORAGE_VERSION: StorageVersion = StorageVersion::new(1);
```